vhdl_parser/src/ast.rs

```diff
@@ -928,12 +928,34 @@ pub struct ComponentSpecification {
     pub component_name: SelectedName,
 }
 
-/// LRM 7.3 Configuration specification
-// @TODO There is also compound
+
+/// LRM 7.3.4 Verification unit binding indication
+#[derive(PartialEq, Debug, Clone)]
+pub struct VUnitBindingIndication {
+    pub vunit_list: Vec<WithPos<Name>>,
+}
+
+
+/// LRM 7.3 Compound Configuration specification
 #[derive(PartialEq, Debug, Clone)]
-pub struct ConfigurationSpecification {
+pub struct CompoundConfigurationSpecification {
     pub spec: ComponentSpecification,
     pub bind_ind: BindingIndication,
+    pub vunit_bind_inds: Vec<VUnitBindingIndication>,
+}
+
+/// LRM 7.3 Simple Configuration specification
+#[derive(PartialEq, Debug, Clone)]
+pub struct SimpleConfigurationSpecification {
+    pub spec: ComponentSpecification,
+    pub bind_ind: BindingIndication,
+}
```

Comment:
Maybe it is unnecessary with two struct here. A simple configuration could just be a compound with and empty vunit_bind_indication. In this way the data model becomes a bit simpler.

Reply:
Right, the two structs seem similar enough. I'll pack them together.